egs/mgb5/s5/local/prepare_data.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+# Copyright 2019 QCRI (Author: Ahmed Ali)
+# Apache 2.0
+
+set -e -o pipefail
+
+
+###
+# The script assumes you have downloaded to the MGB-5 corpus: https://arabicspeech.org/mgb5
+# DB/{dev.tar.gz,train.tar.gz}
+###
+echo "Preparing train and dev data"
+
+if [ ! -d DB ]; then
+  echo "You need to download the MGB-5 first and copy dev.tar.gz and train.tar.gz to DB folder"
```

Comment:
I'd suggest also explicitely check for the files -- if someone creates only the dir, it will generate a lot of errors afterwards

Reply:
done